mlrun/features.py

```diff
@@ -91,32 +93,150 @@ def validator(self, validator):
         self._validator = validator
 
 
+class BasicTypeValidator:
+    def __init__(self):
+        pass
+
+    def check(self, value_type, value):
+        return True, {}
+
+
+class ConvertTypeValidator(BasicTypeValidator):
+    def __init__(self, func):
+        super().__init__()
+        self.func = func
+
+    def check(self, value_type, value):
+        ok, args = super().check(value_type, value)
+        if ok:
+            try:
+                self.func(value)
+            except Exception as err:
+                return (
+                    False,
+                    {"message": err, "type": value_type, "value": value},
```

Comment:
I tried to use the same logic as in original implementation of MinMaxValidator ... but you are right, it makes sense to setup a limits (in case of huge/long values)